CASMCMS-8286 - add console troubleshooting documentation. (#4031)
Cray-HPE/docs-csm: operations/conman/Log_in_to_a_Node_Using_ConMan.md

    cray-console-operator-6cf89ff566-kfnjr
    ```

> 1. (`ncn-mw#`) Set the `XNAME` variable to the component name (xname) of the node whose console you wish to open.

Reviewer: ```suggestion
1. (`ncn-mw#`) Set the `XNAME` variable to the component name (xname) of the node whose console is to be opened.
```

    cray-console-node-1
    ```

> 1. (`ncn-mw#`) Connect to the node's console using ConMan on the `cray-console-node` pod you found.

Reviewer: ```suggestion
1. (`ncn-mw#`) Connect to the node's console using ConMan on the `cray-console-node` pod which was found.
```
